Draytronics Elise V2 - Refactor - Sleep wake key repeat fix,  RBG sleep tweak, readme update. (#23921)
qmk/qmk_firmware: keyboards/draytronics/elise_v2/readme.md

 - If you have one of the provided keymaps flashed, then pressing FN-Tab will enter DFU.
    
Make example for this keyboard (after setting up your build environment):

>     

Reviewer: ```suggestion

```

    make draytronics/elise_v2:default

Flashing example for this keyboard:

    make draytronics/elise_v2:default:flash

> See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with [The QMK Tutorial](https://docs.qmk.fm/#/newbs).

Reviewer: ```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

 - If you have one of the provided keymaps flashed, then pressing FN-Tab will enter DFU.
    

> Make example for this keyboard (after setting up your build environment):

Reviewer: ```suggestion
Make example for this keyboard (after setting up your build environment):

```
